modules/phase_field/src/ics/SmoothCircleBaseIC.C

```diff
@@ -64,7 +65,10 @@ SmoothCircleBaseIC::value(const Point & p)
 RealGradient
 SmoothCircleBaseIC::gradient(const Point & p)
 {
-  Point gradient = Gradient(0.0, 0.0, 0.0);
+  if (_zero_gradient)
+    return 0.0;
```

Comment:
Hmm, Should we return a RealGradient here instead? It's a little odd to see a scalar here.